.cci.jenkinsfile

```diff
@@ -34,7 +34,7 @@ pod(image: imageName + ":latest", kvm: true, cpu: "${cpuCount}", memory: "${memo
         utils.cosaCmd(cosaDir: "/srv", args: "buildextend-live --fast")
     }
 
-    kolaTestIso(cosaDir: "/srv", extraArgs4k: "--no-pxe")
```

Comment:
and now if/when we get to locking down the rpm content in COSA itself CI will block the package bump if something like this happens.